sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxReportSensor.java

```diff
@@ -239,6 +240,21 @@ private boolean saveViolation(Project project, SensorContext context, String rul
           = org.sonar.api.resources.File.fromIOFile(new File(normalPath), project);
```

Comment:
fromIOFile is deprecated, see http://javadocs.sonarsource.org/4.5.2/apidocs/deprecated-list.html. So we should use something like:
```Java
InputFile file= fs.inputFile(fs.predicates().is(new File(filename)));
int maxLine = file.lines();
```
InputFile has method lines.